package.json

```diff
@@ -23,6 +25,6 @@
   "devDependencies": {
     "sinon": "^7.2.2",
     "standard": "^12.0.1",
-    "tap": "^12.1.1"
+    "tap": "15.2"
```

Comment:
ESM test support was added in [12.5](https://github.com/fastify/fastify/pull/2590) but used a special module for it. [v15.0](https://node-tap.org/changelog/#15.0---2021-03-30) dropped that module in favor of native ESM, and 15.2 is the latest minor version of that.